pixels-common/src/test/java/io/pixelsdb/pixels/common/index/MainIndexImplTest.java

```diff
@@ -1,292 +1,292 @@
-        for (SecondaryIndex.Entry entry : entries)
-        {
-            Mockito.verify(entry).setRowId(ArgumentMatchers.anyLong());
-        }
-
-        // Verify cache has 2 rowIds remaining
-        List<Long> cache = TestUtils.getPrivateField(mainIndex, "rowIdCache");
-        Assertions.assertEquals(2, cache.size());
-    }
-
-    @Test
-    public void testPersist_Success()
-    {
-        // Prepare test data
-        mainIndex.putRowIdsOfRg(createRange(100, 200), createLocation(1, 1));
-        mainIndex.putRowIdsOfRg(createRange(300, 400), createLocation(1, 2));
-
-        // Mock successful etcd operation
-        Mockito.doNothing().when(mockedEtcdUtil).putKeyValue(ArgumentMatchers.anyString(), ArgumentMatchers.anyString());
-
-        Assertions.assertTrue(mainIndex.persist());
-
-        // Verify etcd was called twice
-        Mockito.verify(mockedEtcdUtil, Mockito.times(2)).putKeyValue(ArgumentMatchers.anyString(), ArgumentMatchers.anyString());
-    }
-
-    @Test
-    public void testPersistIfDirty_WhenDirty()
-    {
-        mainIndex.putRowIdsOfRg(createRange(100, 200), createLocation(1, 1));
-        Mockito.doNothing().when(mockedEtcdUtil).putKeyValue(ArgumentMatchers.anyString(), ArgumentMatchers.anyString());
-
-        Assertions.assertTrue(mainIndex.persistIfDirty());
-        boolean isDirty = TestUtils.getPrivateField(mainIndex, "dirty");
-        Assertions.assertFalse(isDirty);
-    }
-
-    @Test
-    public void testPersistIfDirty_WhenNotDirty()
-    {
-        Assertions.assertTrue(mainIndex.persistIfDirty());
-        Mockito.verify(mockedEtcdUtil, Mockito.never()).putKeyValue(ArgumentMatchers.anyString(), ArgumentMatchers.anyString());
-    }
-
-    @Test
-    public void testClose_Success() throws IOException
-    {
-        // Data is clean, no need to persist
-        mainIndex.close();
-
-        // Verify no exception thrown
-        Assertions.assertTrue(true);
-    }
-
-    @Test
-    public void testClose_WithDirtyData() throws IOException
-    {
-        mainIndex.putRowIdsOfRg(createRange(100, 200), createLocation(1, 1));
-        Mockito.doNothing().when(mockedEtcdUtil).putKeyValue(ArgumentMatchers.anyString(), ArgumentMatchers.anyString());
-
-        mainIndex.close();
-
-        // Verify dirty flag is cleared
-        boolean isDirty = TestUtils.getPrivateField(mainIndex, "dirty");
-        Assertions.assertFalse(isDirty);
-    }
-
-    @Test
-    public void testClose_PersistFailure()
-    {
-        mainIndex.putRowIdsOfRg(createRange(100, 200), createLocation(1, 1));
-        // Mock putKeyValue throwing exception (causing persist() to fail)
-        Mockito.doThrow(new RuntimeException("ETCD error"))
-                .when(mockedEtcdUtil).putKeyValue(Mockito.anyString(), Mockito.anyString());
-
-        // Verify close() throws IOException
-        Assertions.assertThrows(IOException.class, () -> mainIndex.close());
-    }
-}
\ No newline at end of file
+///*
```

Comment:
Remove this class if you do not need it, or move it to the test source path and make it a JUnit test case.